content/random-linear-combination/en/random-linear-combination.md

```diff
@@ -78,16 +78,16 @@ Now suppose $l_1\neq r_1$ or $l_2 \neq r_2$. The prover still will not be able t
 
 ## Generalizing to $m$ checks
 
-If we have $m$ equality checks, $L_1 = L_1, L_2 = R_2, ..., R_m = R_m$, the verifier could send $m$ random elements $z_1,\dots,z_m$ and the prover could provide $\pi$ such that
+If we have $m$ equality checks, $L_1 = R_1, L_2 = R_2, ..., L_m = R_m$, the verifier could send $m$ random elements $z_1,\dots,z_m$ and the prover could provide $\pi$ such that
 
 $L_1 + L_2z_1 + L_3z_2 + ... L_mz_{m-1} \stackrel{?}{=}R_1 + R_1z_1+R_2z_2+\dots+R_mz_{m-1} + \pi B$
```

Comment:
`R_1z_1+R_2z_2` should be `R_2z_1+R_3z_2`

Reply:
yeah, I will fix it.